workspaces/lightspeed/plugins/lightspeed/package.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "@red-hat-developer-hub/backstage-plugin-lightspeed",
+  "version": "0.1.2",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin",
+    "supported-versions": "1.32.5",
+    "pluginId": "lightspeed",
+    "pluginPackage": "@red-hat-developer-hub/backstage-plugin-lightspeed",
+    "pluginPackages": [
+      "@red-hat-developer-hub/backstage-plugin-lightspeed",
+      "@red-hat-developer-hub/backstage-plugin-lightspeed-backend"
+    ]
+  },
+  "sideEffects": [
+    "./**/*.css"
+  ],
+  "scripts": {
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "lint:check": "backstage-cli package lint",
+    "lint:fix": "backstage-cli package lint --fix",
+    "postpack": "backstage-cli package postpack",
+    "prepack": "backstage-cli package prepack",
+    "start": "backstage-cli package start",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "tsc": "tsc",
+    "prettier:check": "prettier --ignore-unknown --check .",
+    "prettier:fix": "prettier --ignore-unknown --write ."
+  },
+  "dependencies": {
+    "@backstage/core-components": "^0.15.1",
+    "@backstage/core-plugin-api": "^1.10.0",
+    "@backstage/theme": "^0.6.0",
+    "@ianvs/prettier-plugin-sort-imports": "^4.4.0",
+    "@material-ui/core": "^4.9.13",
+    "@material-ui/icons": "^4.11.3",
+    "@material-ui/lab": "^4.0.0-alpha.61",
+    "@mui/icons-material": "^5.15.18",
```

Comment:
Are these MUI dependencies needed if its mostly PF/virtual-assistant? (might be for some integration, I didn't checked)

Reply:
removed all other except @material-ui/core.